### PR TITLE
Switch to Leap 15.5 in CI workflow

### PR DESCRIPTION
**Pull request by baierjan**
Reference: https://progress.opensuse.org/issues/130603

### CODE REVIEW

**Review by perlpunk — COMMENTED** [review]
Why should we remove 15.4 already?

**Comment by baierjan** [comment]
We can leave both until November if that is more desirable. The ticket was suggesting to just replace the versions, so I did.

File: .obs/workflows.yml [hunk 1]
```diff
@@ -16,16 +16,12 @@ pr:
         - target_project: devel:openQA
           target_repository: openSUSE_Tumbleweed
         architectures: [ x86_64 ]
-      - name: openSUSE_Leap_15.4
+      - name: openSUSE_Leap_15.5
```
Comment (kalikiana) [inline .obs/workflows.yml:19]: > To use the most recently released Leap 15.5 as CI container base in os-autoinst and openQA we should adapt that and also run the according OBS checks on Leap 15.4

I would think we want both? Quoting from the ticket.

Comment (baierjan) [reply .obs/workflows.yml:19]: From the very same ticket 😉 
> If it's easier just replace the version

Comment (kalikiana) [reply .obs/workflows.yml:19]: Two votes for just going with 15.5 on the daily so I guess why not

**Review by Martchus — APPROVED** [review]

**Pull request merged**